src/test/java/com/epam/tc/hw5/pages/steps/CommonSteps.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.tc.hw5.pages.steps;
+
+import com.epam.tc.hw5.context.TestContext;
+import com.epam.tc.hw5.pages.HomePage;
+import io.cucumber.java.en.Given;
+import org.openqa.selenium.WebDriver;
+
+public class CommonSteps {
+    private WebDriver webDriver = TestContext.getInstance().getObject(TestContext.WEB_DRIVER);
+
+
+    @Given("I open JDI GitHub site")
+    public void openUrl() {
+        new HomePage(webDriver).openPageUrl();
+    }
+
+    @Given("I login as user \"Roman Iovlev\"")
```

Comment:
It's better to pass user name as a parameter, not as a direct text

Reply:
fixed logic